flickfinder-backend/src/main/java/com/flickfinder/flickfinderbackend/services/MovieFilterService.java

```diff
@@ -0,0 +1,48 @@
+package com.flickfinder.flickfinderbackend.services;
+
+import com.flickfinder.flickfinderbackend.models.Movie;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Flux;
+
+@Service
+public class MovieFilterService {
+
+
+    private final WebClient webClient;
+    private final ApiKeyService apiKeyService;
+
+
+    @Autowired
+    public MovieFilterService(WebClient.Builder webClientBuilder, ApiKeyService apiKeyService) {
+        this.webClient = webClientBuilder.baseUrl("https://api.watchmode.com/v1").build();
```

Comment:
this is a different external api than what we've used for the rest of the application. we've been using TMDB api, which might return results differently than watch mode, so that could affect the use of the other code that has been built. this endpoint I think will be useful for filtering https://developer.themoviedb.org/reference/discover-movie

Reply:
I went and pulled the api url from the Movie service, thank you!